tests/helpers/runif.py

```diff
@@ -83,6 +109,18 @@ def __new__(
             conditions.append(sys.platform == "win32")
             reasons.append("unimplemented on Windows")
 
+        if tpu:
+            conditions.append(not _TPU_AVAILABLE)
+            reasons.append("TPU")
+
+        if horovod:
+            conditions.append(not _HOROVOD_AVAILABLE)
+            reasons.append("Horovod")
+
+        if horovod_nccl:
+            conditions.append(not _HOROVOD_NCCL_AVAILABLE)
+            reasons.append("Horovod with NCCL")
+
```

Comment:
The reasons above are `missing X`

and these are just `X`

choose one 🙂

Reply:
done